syntax = "proto3";

package transport;

// Message de requête
message TransportRequest {
  string start_point = 1;
  string end_point = 2;
}

// Message de réponse
message TransportResponse {
  string transport_type = 1;
  float estimated_time = 2;
}

// Définition du service
service TransportService {
  rpc GetBestTransport(TransportRequest) returns (TransportResponse);
}
